recipes/vs2008_runtime/bld.bat

```diff
@@ -0,0 +1,25 @@
+if "%ARCH%" == "32" (set "ARCH_DIR=x86")
+if "%ARCH%" == "64" (set "ARCH_DIR=amd64")
```

Comment:
Important for determining which directory in WinSxS to look for `vcomp90` libraries of the appropriate architecture.